src/Symfony/Component/Form/Tests/Extension/Validator/DataTransformer/ViewObjectTransformer.php

```diff
@@ -0,0 +1,73 @@
+<?php
+namespace Symfony\Component\Form\Tests\Extension\Validator\DataTransformer;
+
+use Symfony\Component\Form\DataTransformerInterface;
+use Symfony\Component\Form\Exception\TransformationFailedException;
+use Symfony\Component\PropertyAccess\PropertyAccess;
+
+/**
+* Transforms Object to array for view
+*
+* @author Charles J. C. Elling <charles@denumeris.com>
+*/
+class ViewObjectTransformer implements DataTransformerInterface {
```

Comment:
The error you got is `The form's view data is expected to be an instance of class class_name, but is a(n) array.`.
But you say that `we added a view transformer that transforms a instance of class_name to a a(n) array`.

What I understand is that to fix your issue you should either `setting the "data_class" option to null` OR `adding a view transformer that transforms a(n) array to an instance of class_name.` (the opposite of what you're doing).

Am I missing something?